languages/csharp/exercises/concept/inheritance/.docs/introduction.md

```diff
@@ -0,0 +1,59 @@
+In C#, a _class_ hierarchy can be defined using _inheritance_, which allows a derived class (`Car`) to inherit the behavior and data of its parent (general) class (`Vehicle`). If no parent is specified, the class inherits from the `object` class.
```

Comment:
```suggestion
In C#, a _class_ hierarchy can be defined using _inheritance_, which allows a derived class (`Car`) to inherit the behavior and data of its parent class (`Vehicle`). If no parent is specified, the class inherits from the `object` class.
```

I don't think we should define something with two labels ("parent" and "general"). Choose one and reference the other in the after if it's important?

Reply:
We can remove it. Just did :)